TrafficLightsDemo/app_mingw.tcjs

```diff
@@ -1,38 +1,43 @@
 // Windows - MinGW
 let tc = TCF.define(TCF.CPP_TRANSFORM);
-tc.pocoLoc = 'D:\\poco-mingw';
 tc.sources = [
-    'platform:/resource/TrafficLightsDemo/TrafficLightComponent.emx#_rx2SkNXDEei3BJT-OqdJZA'
+    'platform:/resource/TrafficLightsDemo/TrafficLightComponent.emx#_rx2SkNXDEei3BJT-OqdJZA',
+];
+tc.prerequisites = [
+    'platform:/resource/LibHttpServer/httpServerLib.tcjs',
 ];
-tc.topCapsule = 'platform:/resource/TrafficLightsDemo/TrafficLightComponent.emx#_4gz4gNXDEei3BJT-OqdJZA';
-tc.createTargetProject = true;
-tc.targetProject = '/TrafficLightsDemo_target';
 tc.compilationMakeCommand = 'mingw32-make';
 
 tc.compilationMakeType = MakeType.GNU_make;
-tc.targetConfiguration = 'WinT.x64-MinGw-8.1.0';
-tc.compileArguments = '$(DEBUG_TAG) -fexceptions';
+tc.compileArguments = '$(DEBUG_TAG) -fexceptions  -DPOCO_STATIC';
+tc.compileCommand = '$(CC) ';
+tc.createTargetProject = true;
+tc.executableName = 'executable$(EXEC_EXT)';
+tc.linkArguments = '';
```

Comment:
This is the default value for "linkArguments", so this line should be deleted.

Reply:
I will remove it.